contents/blog/the-posthog-array-1-39-0.mdx

```diff
@@ -0,0 +1,108 @@
+---
+date: 2022-08-29
+title: 'Array 1.39.0: Moving person properties onto events'
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ['Release notes', 'Product updates']
+featuredImage: ../images/blog/posthog-array-blog.png
+featuredImageType: full
+excerpt: PostHog 1.39.0 moves person properties onto events, resulting in queries which are up to 400x faster. And that's just for starters!
+---
+
+Want to know more about what we're up to? [Subscribe to HogMail, our newsletter](/newsletter), which we send out every two weeks!
+
+> Running a self-hosted instance? Check out our [Upgrading PostHog guide](/docs/runbook/upgrading-posthog).
+
+## PostHog 1.39.0 release notes
+
+**Release highlights:**
+
+- New: [Person properties have moved on to events](#new-person-properties-have-moved-on-to-events)
+- New: Paul to add a highlight here
+- New: Paul to add a highlight here
+- New: Paul to add a highlight here
+- Improved: Paul to add a highlight here
+- Improved: Paul to add a highlight here
+- Improved: Paul to add a highlight here
+- Apps: JOE to add
+
+### New: Person properties have moved on to events
+
+We've made substantial changes to the way [persons](/manual/persons) and [events](/manual/events) work by moving all persons properties _onto_ events. It's a substantial change, months in the planning, which is critical to PostHog's ability to scale. 
+
+A big benefit of this approach is that it enables significantly faster queries because we no longer have to compare multiple tables when running queries - everything is stored on the event. Some of our internal tests have shown a 400x increase in speed!
```

Comment:
this feels misleading, we saw 400x increase for a single query, this makes me think it's across the fleet 400x

Reply:
Ah, gotcha. Had misunderstood this. 

```suggestion
A big benefit of this approach is that it enables significantly faster queries because we no longer have to compare multiple tables when running queries - everything is stored on the event. One query in our internal tests showed a 400x increase in speed...but this beta will help us understand the improvement in real-world conditions. 
```